<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>2021 Opencvdl Hw1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>1. Image Processing</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_1_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>1.1 Load Image</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_1_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>1.2 Color Seperation</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_1_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>1.3 Color Transformations</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_1_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>420</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>1.4 Blending</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>2. Image Smoothing</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>2.1 Gaussian Blur</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>2.2 Bilateral Filter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>360</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>2.3 Median Filter</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>3. Edge Detection</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>3.3 Sobel Y</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>3.2 Sobel X</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>420</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>3.4 Magnitude</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>3.1 Gaussian Blur</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>510</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Close all windows -----&gt;</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>4. Transformatiion</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>4.3 Rotation, Scaling</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>420</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>4.4 Shearing</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>4.2 Translation</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>4.1 Resize</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>510</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Close all windows</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
